run-server.sh

```diff
@@ -14,4 +14,8 @@ else
     LEVEL="warning"
 fi
 
-/usr/local/bin/gunicorn metadataproxy:app --log-level $LEVEL --workers=2 -k gevent -b $HOST:$PORT
+if [ -z "$WORKERS" ]; then
+    WORKERS="1"
```

Comment:
Can this be `WEB_CONCURRENCY` instead, to follow along in heroku's footsteps? Thats the env var I've seen be the most common across usages.